codespan-reporting/tests/snapshots/term__unicode_spans__rich_no_color.snap

```diff
@@ -6,9 +6,24 @@ error[E01]: cow may not jump during new moon.
   ┌─ moon_jump.rs:1:1
   │
 1 │ 🐄🌑🐄🌒🐄🌓🐄🌔🐄🌕🐄🌖🐄🌗🐄🌘🐄
-  │   ^^ Invalid jump
-  │   -- Cow range does not start at boundary.
-  │   ------ Cow does not start or end at boundary.
+  │ ^^ Invalid jump
```

Comment:
These carets have changed position, due to them lying between character boundaries. If this is a big issue, we can fix this in another pass.

Reply:
agree trying to deal with that in this patch would probably be too much, and there seems to be FIXME's in place.